HW_5.py

```diff
@@ -3,6 +3,31 @@
 # це “буква” + яка вона (велика чи маленька),
 # це “символ”
 #
+def symbol_checker():
+    enter = input("Enter your text ")
+    count = len(enter)
+    for x in enter:
+        if (x.isdigit() == True):
```

Comment:
There is no need to use parentheses here. Also, You can skip `== True` since `if` does this check in the end